proto/profile/local_profile.proto

```diff
@@ -138,6 +138,10 @@ message AppCommand {
       // A subsequent action to start the app will start it with a pristine runtime state.
       // This command will purge ALL volumes used by the application.
       COMMAND_PURGE = 2;
+      // Application instance will be shut down.
+      COMMAND_STOP = 3;
+      // Application instance will be started.
+      COMMAND_START = 4;
```

Comment:
Seems like there is no request for these commands (and their semantics would be odd when the device has intermittent connectivity to the controller.)

So I think we should drop these from the PR.